core/consensus/component.go

```diff
@@ -382,8 +383,8 @@ func (c *Component) Participate(ctx context.Context, duty core.Duty) error {
 		return nil // No eager consensus for potential no-op aggregation duties.
 	}
 
-	if !c.timerFunc(duty).Type().Eager() {
-		return nil // Not an eager start timer, wait for Propose to start.
+	if featureset.Enabled(featureset.ConsensusParticipate) {
+		return nil // Wait for Propose to start.
```

Comment:
Shouldn't this check be flipped?

i.e. if `ConsensusParticipate` is disabled, don't proceed.

Reply:
Fixed